KS.Fiks.Arkiv.Models.V1/Meldingstyper/FiksArkivV1PayloadHelper.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+
+namespace KS.Fiks.Arkiv.Models.V1.Meldingstyper
+{
+    public class FiksArkivV1PayloadHelper
```

Comment:
Ligger allerede under ..V1... så tenker PayloadHelper er bra nok. Når det kommer v2 vil de bli skilt på namespace uansett

Reply:
Sure. Da bør vi gjøre det samme for den andre klassen der